lib/class-wp-rest-plugins-controller.php

```diff
@@ -61,11 +72,63 @@ public function delete_item( $request ) {
 	}
 
 	public function prepare_item_for_response( $item, $request ) {
-
+		return $item;
 	}
 
 	public function get_item_schema() {
-
+		$schema = array(
+			'$schema'              => 'http://json-schema.org/draft-04/schema#',
+			'title'                => 'plugin',
+			'type'                 => 'object',
+			'properties'           => array(
+				'Name'        => array(
```

Comment:
Here's what I see:

```
["wp-api-menus-endpoints/plugin.php"]=>
    array(11) {
    ["Name"]=>
    string(28) "WP REST API - Menu Endpoints"
    ["PluginURI"]=>
    string(48) "https://github.com/WP-API/wp-api-menus-endpoints"
    ["Version"]=>
    string(5) "0.1.0"
    ["Description"]=>
    string(34) "Menu endpoints for the WP REST API"
    ["Author"]=>
    string(16) "WP REST API Team"
    ["AuthorURI"]=>
    string(17) "http://wp-api.org"
    ["TextDomain"]=>
    string(0) ""
    ["DomainPath"]=>
    string(0) ""
    ["Network"]=>
    bool(false)
    ["Title"]=>
    string(28) "WP REST API - Menu Endpoints"
    ["AuthorName"]=>
    string(16) "WP REST API Team"
}
```

It'd be worth diving in a bit more to see what the distinction is.